src/Microsoft.Identity.Client/AppConfig/AbstractApplicationBuilder.cs

```diff
@@ -104,69 +122,96 @@ public T WithTelemetryCallback(TelemetryCallback telemetryCallback)
+        /// or a domain associated with this Azure AD tenant, in order to sign-in user of a specific organization only</param>
+        /// <returns>The builder to chain the .With methods</returns>
         public T WithTenantId(string tenantId)
         {
             Config.TenantId = GetValueIfNotEmpty(Config.TenantId, tenantId);
             return (T)this;
         }
 
         /// <summary>
+        /// Enables/disable logging of Personally Identifiable Information. See https://aka.ms/msal-net-logging
         /// </summary>
-        /// <param name="enablePiiLogging"></param>
-        /// <returns></returns>
+        /// <param name="enablePiiLogging">Boolean used to enable/disable logging of 
+        /// Personally Identifiable data (PII) data. 
+        /// PII logs are never written to default outputs like Console, Logcat or NSLog
+        /// Default is set to <c>false</c>, which ensures that your application is compliant with GDPR. 
+        /// You can set it to <c>true</c> for advanced debugging requiring PII</param>
+        /// <returns>The builder to chain the .With methods</returns>
         public T WithEnablePiiLogging(bool enablePiiLogging)
         {
             Config.EnablePiiLogging = enablePiiLogging;
             return (T)this;
         }
 
         /// <summary>
+        /// Enables you to configure the level of logging you want. See https://aka.ms/msal-net-logging
+        /// The default value is <see cref="LogLevel.Info"/>.
+        /// Setting it to <see cref="LogLevel.Error"/> will only get errors
+        /// Setting it to <see cref="LogLevel.Warning"/> will get errors and warning, etc..
         /// </summary>
-        /// <param name="logLevel"></param>
-        /// <returns></returns>
+        /// <param name="logLevel">Desired level of logging</param>
+        /// <returns>The builder to chain the .With methods</returns>
         public T WithLoggingLevel(LogLevel logLevel)
         {
             Config.LogLevel = logLevel;
             return (T)this;
         }
 
         /// <summary>
+        /// Flag to enable/disable logging to platform defaults. See https://aka.ms/msal-net-logging
+        /// In Desktop/UWP, Event Tracing is used. In iOS, NSLog is used.
+        /// In android, logcat is used. The default value is <c>false</c>
         /// </summary>
-        /// <param name="enabled"></param>
-        /// <returns></returns>
+        /// <param name="enabled">Boolean telling if default logging is
+        /// enabled or not</param>
+        /// <returns>The builder to chain the .With methods</returns>
         public T WithDefaultPlatformLoggingEnabled(bool enabled)
         {
             Config.IsDefaultPlatformLoggingEnabled = enabled;
             return (T)this;
         }
 
         /// <summary>
+        /// Sets application options, which can, for instance have been read. See https://aka.ms/msal-net-application-configuration.
+        /// from configuration files.
```

Comment:
the aka.ms link need to be moved to after "from configuration files" on line 211. the order is not correct and it's in-between the sentence.